dockstore-webservice/src/main/java/io/dockstore/webservice/core/Version.java

```diff
@@ -242,6 +242,14 @@ public abstract class Version<T extends Version> implements Comparable<T> {
     @ApiModelProperty(value = "The aggregated metrics for executions of this version, grouped by platform", position = 26)
     private Map<Partner, Metrics> metricsByPlatform = new EnumMap<>(Partner.class);
 
+    @Column()
```

Comment:
Was initially thinking you could do this with a single "dirty" boolean. 
i.e. dirty if metrics have been submitted after an aggregation

but this may be useful information in general to track, so good call!